_data/nav.yml

```diff
@@ -610,16 +610,14 @@
     - url: "#New features"
     - url: "#Updates and enhancements"
     - url: "#Known issues"
-  - url: "premium-plugins-errata509"
-    pages:
-    - url: "#Overview"
-    - url: "#Tiny Comments"
-    - url: "#PowerPaste"
   - url: "release-notes509"
     pages:
     - url: "#Overview"
     - url: "#New features"
     - url: "#Updates and enhancements"
+    - url: "#Errata: TinyMCE 5.0.9 Premium Plugins"
+    - url: "#Tiny Comments"
```

Comment:
I'd remove "Tiny Comments" and "PowerPaste" here, as we normally only include top-level headings.

Reply:
Amended Commit to fix issue.